### DISCUSSION TITLE
osu!lazer broken mobile input after update

### DISCUSSION THREAD

**Discussion by anton2007** [ppy/osu#15964] [Q&A] [answered]
After an update about a month ago something happened with the mobile soft keyboard input on all my Android devices (Huawei Honor 9S, Huawei Y3II,  Samsung Galaxy M12).

When I updated and tried to login I wasn't able to type any symbol on the soft keyboard, but in 2021.1028 version the soft keyboard still works.

**Comment by anton2007** [comment]
Peppy, you can say what you did with soft keyboard on Android? (Probably osu!framework related)

**Comment by bdach** [comment, marked as answer]
https://github.com/ppy/osu/issues/8264
